client/src/components/SupportTools/RoundsTable.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { Button } from '@blueprintjs/core'
+import { toast } from 'react-toastify'
+
+import { Confirm, useConfirm } from '../Atoms/Confirm'
+import { IRound, useReopenCurrentRound, useUndoRoundStart } from './support-api'
+import { StyledTable } from '../Atoms/Table'
+import StatusTag from '../Atoms/StatusTag'
+
+interface IProps {
+  electionId: string
+  rounds: IRound[]
+}
+
+const RoundsTable = ({ electionId, rounds }: IProps) => {
+  if (rounds.length === 0) {
+    return (
+      <StyledTable style={{ tableLayout: 'auto' }}>
+        <thead>
+          <tr>
+            <th>Round</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Round 1</td>
+            <td>
+              <StatusTag>Not started</StatusTag>
+            </td>
+          </tr>
+        </tbody>
+      </StyledTable>
+    )
+  }
+
+  return (
+    <StyledTable style={{ tableLayout: 'auto' }}>
+      <thead>
+        <tr>
+          <th>Round</th>
+          <th>Status</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        {rounds.map((round, i) => (
+          <tr key={round.id}>
+            <td>Round {round.roundNum}</td>
+            <td>
+              {round.endedAt ? (
+                <StatusTag intent="success">Completed</StatusTag>
+              ) : (
+                <StatusTag intent="warning">In progress</StatusTag>
+              )}
+            </td>
+            <td>
+              {i === rounds.length - 1 && (
+                <LastRoundAction electionId={electionId} round={round} />
+              )}
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </StyledTable>
+  )
+}
+
+interface ILastRoundActionProps {
+  electionId: string
+  round: IRound
+}
+
+const LastRoundAction = ({ electionId, round }: ILastRoundActionProps) => {
+  const { confirm, confirmProps } = useConfirm()
+  const reopenCurrentRound = useReopenCurrentRound()
+  const undoRoundStart = useUndoRoundStart()
+
+  if (round.endedAt) {
+    return (
+      <>
+        <Button
+          onClick={() =>
+            confirm({
+              title: 'Confirm',
+              description: `Are you sure you want to reopen Round ${round.roundNum}?`,
+              yesButtonLabel: 'Reopen',
+              onYesClick: async () => {
+                await reopenCurrentRound.mutateAsync({ electionId })
+                toast.success(`Reopened Round ${round.roundNum}`)
+              },
+            })
+          }
+        >
+          Reopen
+        </Button>
+        <Confirm {...confirmProps} />
+      </>
+    )
+  }
+  return (
```

Comment:
I know there are additional conditions that determine whether or not a round start can be undone, e.g. whether or not audit boards have already been opened. Rather than hiding/disabling the button in these cases, I'm just relying on error toasts, which include helpful error messages from the server, e.g. 

https://github.com/votingworks/arlo/blob/70024cc33f5715da262774eb92e7bd771ad4e715/server/api/support.py#L420-L422